source/systemadministration/install-on-kvm/install.rst

```diff
@@ -594,6 +594,39 @@ adminPC: Erstellen eines ssh-Aliases zum Server
 	</p>
 ..
 
+server: Anpassen der Linbo-Konfiguration
+----------------------------------------
+
+Zum Abschuss der Installation des **servers** müssen noch zwei Konfigurationsparameter angepasst werden.
+
+**Aktivieren des Bittorrent Dienstes**
+
+.. code-block:: console
+
+	# /etc/default/bittorrent
+	START_BTTRACK=1
```

Comment:
@morpweb Bist du sicher, dass das nötig ist? Es gibt ja auch noch https://github.com/linuxmuster/linuxmuster-linbo/blob/master/debian/linbo-bittorrent.default Vielleicht weiß @HappyBasher mehr.

Reply:
@zefanja Bin sicher. Damit Bittorrent funktioniert, muss der Parameter derzeit "von Hand" gesetzt werden. Siehe z.B. hier: https://ask.linuxmuster.net/t/linbo-verteilt-keine-images/2006/3 .

Reply:
@morpweb Könntest du dazu bitte einen Bugreport erstellen (hier: https://github.com/linuxmuster/linuxmuster-linbo/issues)? In dem verlinkten Beitrag scheint es aber vorher auch mit Bittorrent geklappt zu haben, bevor das Problem auftrat.

Reply:
@zefanja Kann ich gern machen. Bin nur nicht sicher, ob es wirklich ein Bug ist. Es ist eher ein historisches Problem. ;-) Früher war bei Linbo Standard, Images per Multicast zu verteilen. Deshalb wurde der Bittorrent Service nicht aktiviert. Ist wohl eher ein Hinweis für die 7er Version ...  Habe Ticket erstellt.

Reply:
Danke dir!